python/Ganga/GPIDev/Adapters/IGangaFile.py

```diff
@@ -71,6 +72,15 @@ def put(self):
         """
         raise NotImplementedError
 
+    def copyTo(self, targetPath):
+        """
+        Copy a the file to the local storage using the get mechanism
+        Args:
+            targetPath (str): Target path where the file is to copied to
+        """
+        self.localDir = targetPath
```

Comment:
This is the problem currently with the `IGangaFile` interface not being defined properly until now. I would like to say that `get` always copied from the 'remote location' to `localDir`. However, if that's not the case then that should probably be addressed first/as well. In any case, `copyTo` shouldn't be changing the File object, i.e. the `localDir` should be changed back to what it was originally. What is happening here is conceptually more a 'move'. I would say it's also worth checking if the file is already in the `localDir` as then it can just do a local file copy instead of calling `get`.